templates/ironicneutronagent/config/ironic_neutron_agent.ini

```diff
@@ -0,0 +1,31 @@
+
+[DEFAULT]
+auth_strategy = keystone
+
+[keystone_authtoken]
+project_domain_name=Default
+user_domain_name=Default
+project_name=service
+username={{ .ServiceUser }}
+www_authenticate_uri={{ .KeystonePublicURL }}
+auth_url={{ .KeystoneInternalURL }}
+auth_type=password
+
+[service_catalog]
+auth_type=password
+auth_url={{ .KeystoneInternalURL }}
+username={{ .ServiceUser }}
+user_domain_name=Default
+project_name=service
+project_domain_name=Default
+
+[ironic]
+auth_type=password
+auth_url={{ .KeystoneInternalURL }}
+username={{ .ServiceUser }}
+user_domain_name=Default
+project_name=service
+project_domain_name=Default
+
+[agent]
+report_interval=30
```

Comment:
Yes it can be tuned using the `customServiceConfig` interface. For example:
```
spec:
  customServiceConfig: |
    [agent]
    report_interval=60
```
The `merge_config_dir` function uses `crudini --merge` to update the config to include what is in customServiceConfig during init.